docs/dasharo-tools-suite/releases.md

```diff
@@ -5,298 +5,11 @@ for Dasharo Tools Suite.
 
 ## **Latest release**
 
-**Versions newer than `v1.2.11` are not published here.**{ .md-banner--warning }
-Instead, they can be found on `meta-dts` GitHub repository, inside `Releases`
+Latest release can be found on `meta-dts` GitHub repository, inside `Releases`
 tab. The button below will take you there there.
```

Comment:
@DaniilKl two times `there`. Could you fix this?

Reply:
@TomaszAIR, fixed here eb415381108dee5438e590f8fefbdfb89fe94838.